draft-ietf-ace-cwt-proof-of-possession.xml

```diff
@@ -480,15 +487,15 @@
     <section anchor="Security" title="Security Considerations">
       <t>
         All the security considerations that
-        are discussed in <xref target="RFC8392"/> also apply here.
+        are discussed in <xref target="CWT"/> also apply here.
         In addition, proof of possession introduces its own unique security issues.
 	Possessing a key is only valuable if it is kept secret.
 	Appropriate means must be used to ensure that unintended parties
 	do not learn private key or symmetric key values.
       </t>
       <t>
 	Applications utilizing proof of possession SHOULD also utilize audience restriction,
-	as described in Section 4.1.3 of <xref target="JWT"/>,
+	as described in Section 4.1.3 of <xref target="CWT"/>,
```

Comment:
The audience section number in CWT is 3.1.3.  I'll merge this anyway and then fix it before publication.